lib/attachments/imageResizer.js

```diff
@@ -23,106 +15,68 @@ function generateThumb(options) {
       toPath = options.toPath,
       imageHeight = options.imageHeight,
       imageWidth = options.imageWidth,
-      minify = options.minify;
-  return readImage(fromPath).then(resizeImage({
+      _options$minify = options.minify,
+      minify = _options$minify === void 0 ? false : _options$minify;
```

Comment:
Kannte den Ausdruck `void 0` garnicht. Geht das nicht auch in einem einfacheren Ausdruck?
`minify = options.minify || false`

Reply:
Sorry, hab vergessen zu schreiben, dass in den lib Ordner das generierte Zeug von babel ist. Also das hat babel so erzeugt. Wurde nicht von uns so geschrieben :)

Reply:
Die Datei ist ein Build-Artefakt, die habe ich mal geskippt, da passiert noch mehr so komisches Zeug.